### DISCUSSION TITLE
Archiving this repo / discussions

### DISCUSSION THREAD

**Discussion by ahpook** [community/OSPO#136] [Announcements]
Hi, things have been pretty quiet here and to avoid a cobwebby spam trap I'm going to archive the repo on 2024-04-01 and remove it a month later on 2024-05-01. We are focusing our community-building energy on the github/github-ospo repo, which has a [project board](https://github.com/orgs/github/projects/12731) for the OSPO Actions plus Issues enabled for content and code related questions. Thanks to all who participated here!

**Comment by ljharb** [comment]
Why remove it? That means nothing will be searchable. Can’t it be left archived forever?

**Reply by ahpook** [reply]
I just dislike loose ends, I suppose. :) But as a practical matter most of the discussion is about the beta metrics dashboard which we EOL'ed, so I'm not sure how valuable it really is.

**Reply by ljharb** [reply]
[Cool URLs don't change](https://www.w3.org/Provider/Style/URI) :-)
